shared/chat/conversation/messages/wrapper/exploding-meta/index.js

```diff
@@ -44,8 +45,9 @@ class ExplodingMeta extends React.Component<Props, State> {
     if (this.props.exploded && !prevProps.exploded) {
       this.setState({mode: 'boom'})
       SharedTimer.removeObserver(this.props.messageKey, this.sharedTimerID)
+      this.sharedTimerKey = this.props.messageKey
```

Comment:
we should stash what was actually passed and not this prop based thing since that could change and our unregister could be incorrect